lib/editor-settings.php

```diff
@@ -81,3 +81,65 @@ function gutenberg_extend_post_editor_settings( $settings ) {
 	return $settings;
 }
 add_filter( 'block_editor_settings', 'gutenberg_extend_post_editor_settings' );
+
+/**
+ * Initialize a block-based editor.
+ *
+ * @param array $settings {
+ *      Elements to initialize a block-based editor.
+ *
+ *      @type array  $preload_paths        paths need to be preloaded.
+ *      @type string $editor_name          editor name.
+ *      @type string $editor_script_handle editor script handle.
+ *      @type string $initializer_name     editor initialization function name.
+ *      @type array  $editor_settings      editor settings.
+ * }
+ * @return void
+ */
+function gutenberg_initialize_editor( $settings ) {
+
+	$defaults = array(
+		'preload_paths'        => array(),
+		'editor_name'          => 'widgets_editor',
+		'editor_script_handle' => 'edit-widgets',
```

Comment:
Why is "widgets" the default? Seems quite odd.

Reply:
It is simply my first need . What is preferable then ? Empty strings but it is required in the code behind.